CONTRIBUTING.md

```diff
@@ -23,3 +23,21 @@ changes, feel free to ping @gwyneplaine
 * Please [follow our established coding conventions](https://github.com/keystonejs/keystone/wiki/Coding-Standards)
 (with regards to formatting, etc)
 * All new features and changes need documentation.
+
+## Running the docs website
+
+You can run the website in a local environment using the following steps:
+
+```sh
+git clone https://github.com/JedWatson/react-select
+cd react-select
+yarn
+yarn build
+cd docs
```

Comment:
This feels wrong to me since the script itself is describe as
`"start": "cd docs && yarn start",`